src/annotator-content-analysis.js

```diff
@@ -1,54 +1,136 @@
-function annotatorCustomEditor(options){
+function annotatorContentAnalysis(options){
   return {
     start: function() {
       this.options = options;
-      this.getDataPayload();
+      this._elements();
+      this.getTemplates();
+      this.getData();
     },
 
-    // ajax get to retrieve data
-    getDataPayload: function(){
-      $.get(this.options.endpoint, (data) => {
-        this.configureData(data);
-      })
+    _elements: function() {
```

Comment:
to set survey nodes as empty arrays. this is just a generic method for DOM elements or vars. make sense?